client/config/environment.js

```diff
@@ -65,10 +64,10 @@ module.exports = function(environment) {
     ENV.contentSecurityPolicy = {
       'default-src': "'none'",
       // add the domain here like analytics.google.com or data: or 'self'
-      'script-src':  "'self' 'unsafe-eval' 'https://www.google.com' ''",
+      'script-src':  "'self' 'unsafe-eval' 'unsafe-inline' 'https://www.google.com' 'www.google-analytics.com'",
```

Comment:
You're right, from the addon docs:

> During the build process, this will render the contents of those files directly inline with <script>

Is this the best strategy if it opens up XSS attacks?  I believe it is recommended not to inline JS in general, right?